### ISSUE TITLE
Problems with MGS2Setup (The solutions doesn't work)

### ISSUE THREAD

**Issue by YisusKuraitsu** [VFansss/mgs2-v-s-fix#52]
Hello, I saw other answers to this, like the UAC in the bin folder and the 'debug mode' in there two ways and I tried but it doesn't work. When I execute, the game still shows the problem "Can't do a sh*t" saying that "Some operations has been locked by operation system..." and the two solutions. What can I do? Thanks.
[MGS2_VFix_debug.txt](https://github.com/VFansss/mgs2-v-s-fix/files/2346799/MGS2_VFix_debug.txt)

**Comment by VFansss** [comment]
What operating system do you have?

**Comment by YisusKuraitsu** [comment]
Windows 10

**Comment by VFansss** [comment]
Ok, here's the deal: Windows or your antivirus is blocking the execution of the fix in its preliminary actions: install the official 2.0 Path.

The fastest way to solve this is:

* [download the patch here](https://github.com/VFansss/mgs2-v-s-fix/blob/master/mgs2%20v's%20fix/Resources/1_homemade_patcher.zip)

* extract all 3 files inside the "/bin" game directory. Overwrite files, if a prompt appear.

* Launch **AreaChecker.exe**, than **VOXPatch.exe**

* Create a file called "**_fixed_exe_applied.sss**" (without quotes, but with underscore) inside the "/bin" game folder. Mind the extension.

Then you have manually applied what V's Fix can't apply because something is blocking it.

Then launch again the fix, and it should continue normally.

**Comment by YisusKuraitsu** [comment]
It worked! Many thanks!

**Comment by VFansss** [comment]
Please send me the **last.log** file you can find inside the "/bin" game folder.

Also, try to start manually "mgs2_sse.exe".

You can also check if the game is starting using compatibility flags, clicking on "mgs2_sse.exe" with the right mouse button, select "Properties", than "Compatibility" and check if "Start with admin rights" and "Windows XP Compatibility" is flagged.

My guess is that windows is blocking the execution of the .exe. I don't know why your system is so restrictive about it...

**Comment by YisusKuraitsu** [comment]
I tried that but nothing happens, I can go to the settings of the game now in the VR interface, but when I press "Start Game" nothing happens. Here is my **last.log**
[last.log](https://github.com/VFansss/mgs2-v-s-fix/files/2362628/last.log)

**Comment by VFansss** [comment]
Oh god, I totally forgot to reply to you :sob: 

Are you still having issues? :cry:

**Comment by YisusKuraitsu** [comment]
Don't worry and yes, I still have the last problem that I mentioned

**Comment by VFansss** [comment]
If you want, we could solve the issue.

> I can go to the settings of the game now in the VR interface

So from what I can see (even from the last.log file) the game is actually starting, am I right?

Please open the fix, press SAVE and then send me again the MGS2_VFix_debug.txt file on your desktop, so I could see your PC info and your actual Fix config.

My suspect is that something is blocking the Fix from its normal behaviour, and the fix for the controller isn't actually extracted.

Do you have any anti-virus?

Also, you could try to uninstall the game and install it somewhere else different from "Program Files".

**Comment by YisusKuraitsu** [comment]
Haha, one problem solved and one problem new. I re-installed and put the fix in a new folder away from Program Files. Now I can open the game. But, now it's like I had never put any fix. The problem of the unseen textures keep going. Here's my last **MGS2_VFix_Debug.txt**. I tried with the AreaChecker, VOXPatch and _fixed_exe_applied.sss too, but I can't see any texture. Well, at least the game opens.
[MGS2_VFix_debug.txt](https://github.com/VFansss/mgs2-v-s-fix/files/2476112/MGS2_VFix_debug.txt)

**Comment by VFansss** [comment]
> I tried with the AreaChecker, VOXPatch and _fixed_exe_applied.sss too

???

> Here's my last MGS2_VFix_Debug.txt.

Please read better the [DEBUG MODE](https://github.com/VFansss/mgs2-v-s-fix/wiki/Troubleshooting-&-Debug-mode#debug-mode) page.

While you have debug mode activated, open the fix, press OPTION, then press 'SAVE' and then the log will be filled by information of all kind.

Instead, you sent me a file where you started the fix and then started the game, leaving me without enough info.

Please do that and send me again another debug file!

> but I can't see any texture

Probably is a VGA related issue. Please send me the right MGS2_VFix_debug.txt debug file **AND** the last.log file you can find inside the game folder.

**Comment by VFansss** [comment]
The solution is very easy: you didn't select the right VGA from your NVidia Control Panel.

I will explain it better: your laptop decide itself if the game must run on the integrated vga (The Intel HD) or the dedicated one ( your Nvidia GTX 1050 Ti).

If you select a VGA from the fix, you have to be sure that your laptop is following the same dictate.

From the log, I saw that you have selected the 1050 Ti, but the game is starting using the Intel HD, and has graphical glitches.

So, what's the solution? There's two of them:

* ( Easier ) - Make the game starting with the integrated VGA - Just select the 'Intel HD' from the V's Fix and press SAVE. Obviously, the game will run with the integrated VGA. Your Intel HD VGA is more that enough to make it working good, but...

* ( "Harder") - Open your VGA control panel and make the game executable ( mgs2_sse.exe) to run with "higher quality", "maximum performance", or something. In this way the game will run on the 1050 Ti. I can't precisely tell you how to do that because I don't have an NVidia, but I bet that you do "right click on desktop -> Open NVidia control panel" and set something! [In my old AMD control panel I can set VGA affinity through this panel](https://imgur.com/xXxgoNQ (NB: "Prestazioni Elevate" means "High performance", in italian)

Tell me if you have issues 😄

**Comment by VFansss** [comment]
> For the patience and your great work with this fix.

There were common issues. Maybe I could improve the fix making it more aware of these issues.

Definitively on the "ToDo" List for the next version.

Thanks you too for the patience, but this month was horrific ☠️ 

> Man, seriously, I can't express my gratitude enough.

The best way I know is play and having fun 😄

**Comment by VFansss** [comment]
I'm going to add to the wiki a separate paragraph for solving this issue with VGA Drivers.

I've also added a warning to the 'Resolution' tab, an info should catch up the first time the fix is running (only if it detect more VGAs on the system), and I've enhanced the debugger so it should be much more easier to detect this kind of situation.

Also, now the V's Fix detect if the game has started with a "wrong VGA adapter", and prompts itself the solution if is the case. Also, it point you directly on the right wiki page, so the solution is even more rapid to be found.

Thanks for reporting me this, and have fun 😄
